db/frontend.go

```diff
@@ -299,6 +360,16 @@ func DeleteSubscription(userID uint64, network string, eventName types.EventName
 	return err
 }
 
+func DeleteSubscriptionBatch(userID uint64, network string, eventName types.EventName, eventFilter []string) error {
+	name := string(eventName)
+	if network != "" && !types.IsUserIndexed(eventName) {
+		name = strings.ToLower(network) + ":" + string(eventName)
+	}
+
+	_, err := FrontendWriterDB.Exec("DELETE FROM users_subscriptions WHERE user_id = $1 and event_name = $2 and event_filter = ANY($3)", userID, name, pq.Array(eventFilter))
```

Comment:
Please use uppercase AND, like in the other statements.

Reply:
changed it now to upper case - also in the sql above and blow.